src/consts/index.ts

```diff
@@ -27,3 +27,16 @@ export const GCS_PRIVATE_KEY = process.env.GCS_PRIVATE_KEY || '';
 export const GCS_BASE_URL = 'https://storage.googleapis.com';
 
 export * from './upload';
+
+const mustBePresent: { [key: string]: string | number } = {
```

Comment:
Can just remove this portion because it is quite hard to manage different sets of env variables for heroku and app engine. Lines 31 to 42 can be deleted.

Reply:
Removed